circuit/src/gadgets/lt_eq.rs

```diff
@@ -73,7 +84,20 @@ impl<F: FieldExt> LessEqualChip<F> {
 
 			let x_next_exp = v_cells.query_advice(x, Rotation::next());
 
-			vec![s_exp * ((x_exp + n_shifted_exp - y_exp) - x_next_exp)]
+			vec![
+				// (x + n_shifted - y) - z == 0
+				// n_shifted value is equal to smallest 253 bit number.
+				// Because of that calculations will be done in between 252 to 254 bit range.
+				// That range can hold 252 bit number calculations without overflowing.
```

Comment:
```suggestion
				// Because of that calculations will be done in between the 252 to 254-bit range.
				// That range can hold 252-bit number calculations without overflowing.
```